erpnext/hr/doctype/training_event/training_event.js

```diff
@@ -17,5 +17,37 @@ frappe.ui.form.on('Training Event', {
 				frappe.set_route("List", "Training Feedback");
 			});
 		}
+	},
+	onload: function(frm) {
+		var params = getSearchParameters();
```

Comment:
I am not sure if this is the correct way to pass parameters from email.
Let's see what @rmehta has to say.